rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -4,12 +4,11 @@
 import sys
 import time
 
+sys.path.insert(1, os.path.dirname(__file__)) # noqa: E261
 from deoplete_jedi import cache, profiler, utils, worker
```

Comment:
I think it should be moved to `deoplete_jedi/__init__.py`, but then deoplete does not find it anymore?!